features/git-extract/cherry_pick_conflict/with_remote_origin/with_open_changes.feature

```diff
@@ -79,12 +74,19 @@ Feature: git extract: resolving conflicts between main branch and extracted comm
     And I end up on the "refactor" branch
     And I again have an uncommitted file with name: "uncommitted" and content: "stuff"
     And now I have the following commits
-      | BRANCH   | LOCATION         | MESSAGE         | FILE NAME        |
-      | main     | local and remote | main commit     | conflicting_file |
-      | feature  | local            | feature commit  | feature_file     |
-      |          |                  | refactor commit | conflicting_file |
-      | refactor | local and remote | main commit     | conflicting_file |
-      |          |                  | refactor commit | conflicting_file |
+      | BRANCH   | LOCATION         | MESSAGE         |
+      | main     | local and remote | main commit     |
+      | feature  | local            | feature commit  |
+      |          |                  | refactor commit |
+      | refactor | local and remote | main commit     |
+      |          |                  | refactor commit |
+    And now I have the following committed files
+      | BRANCH   | NAME             | CONTENT          |
+      | main     | conflicting_file | main content     |
+      | feature  | conflicting_file | refactor content |
+      | feature  | feature_file     |                  |
```

Comment:
Drying up should be done in a separate PR now that you are using Mortadella for these tables